### ISSUE TITLE
Stay signed in is not working

### ISSUE THREAD

**Issue by kdcllc** [JudahGabriel/Chavah#80]
Stay signed in functionality is not enabled. Once user signed it and Stay signed in is checked the user information is not saved upon the browser close. The sign screen is promoted again.

**Comment by JudahGabriel** [comment]
I'm not seeing that here. I stay signed in for weeks at a time.

**Comment by kdcllc** [comment]
![signin](https://user-images.githubusercontent.com/13120940/44352282-e8469680-a471-11e8-9fbb-84601c974f5e.gif)
As you can see once I close the browser the state of being sign in doesn't exist any longer.

**Comment by kdcllc** [comment]
Yes it keeps me signed in. I am confused why we have `http` without redirection to `https` since we have this in `Startup.cs`
```
      app.UseHttpsRedirection();
```
My understanding was that it would force the redirect to `https` scheme.

**Comment by JudahGabriel** [comment]
Good question. It would appear that's not working.

A little backstory:

When we switched to HTTPS about 2 years ago, I used the [RequireHttps] attribute to force HTTPS. Several listeners complained they couldn't load Chavah. As I investigated, the problem turned out to be they were all running on Windows XP. (cue hisses and boos)

Apparently Windows XP doesn't honor LetsEncrypt certificates, which is what I'm using for HTTPS. So...I didn't require HTTPS.

But now I'm willing to just drop Windows XP support altogether. The web is moving towards HTTPS-only, and Chavah should as well. So, we should make sure HTTPS redirection is working.

**Comment by kdcllc** [comment]
We might need to add this in `ConfigureServices`:
```
            services.AddHttpsRedirection(options => options.RedirectStatusCode = StatusCodes.Status308PermanentRedirect );
```

**Comment by JudahGabriel** [comment]
Max fixed this in the dev branch. I've merged the dev branch into master, and it's now deployed.

**Comment by kdcllc** [comment]
@JudahGabriel it seems that you might need to force the https redirection inside azure like so 
![image](https://user-images.githubusercontent.com/13120940/46760136-2f494f00-cc9f-11e8-87ad-6d4ce0e18247.png)

**Comment by JudahGabriel** [comment]
Chavah itself is not running in an Azure web app, but inside an Azure virtual machine, so there's no such setting.

However, just adding [RequireHttps] on the HomeController is enough to redirect folks. I've published that and it appears to work.
